packages/ui/src/components/SelectableCard/__tests__/index.test.tsx

```diff
@@ -4,6 +4,9 @@ import { renderWithTheme, shouldMatchEmotionSnapshot } from '@utils/test'
 import { describe, test, vi } from 'vitest'
 import { SelectableCard } from '..'
 
+const illustration =
+  'https://ultraviolet.s3.fr-par.scw.cloud/illustrations/products/appleSilicon/apple-silicon-wire.svg'
+
```

Comment:
this might be risky because the link might die someday and we won't understand why the test fail. Maybe import locally a svg that you will use in your test instead